internal/diskutils/diskutil.go

```diff
@@ -47,245 +36,77 @@ type Command interface {
 	CombinedOutput() ([]byte, error)
 }
 
-// IDPathNotFoundError indicates that a symlink to the device was not found in /dev/disk/by-id/
-type IDPathNotFoundError struct {
-	DeviceName string
-}
-
-func (e IDPathNotFoundError) Error() string {
-	return fmt.Sprintf("IDPathNotFoundError: a symlink to  %q was not found in %q", e.DeviceName, DiskByIDDir)
-}
-
 // BlockDevice is the a block device as output by lsblk.
-// All the fields are lsblk columns.
-
 type BlockDeviceList struct {
 	BlockDevices []BlockDevice `json:"blockdevices"`
 }
 
 type BlockDevice struct {
 	Name       string        `json:"name"`
-	Rotational bool          `json:"rota"`
 	Type       string        `json:"type"`
 	Size       int64         `json:"size"`
+	PathByID   string        `json:"id-link,omitempty"`
 	Model      string        `json:"model,omitempty"`
 	Vendor     string        `json:"vendor,omitempty"`
 	ReadOnly   bool          `json:"RO,omitempty"`
```

Comment:
I'd personally lower case these 2 fields, just to keep consistency with the rest of the field names. I don't mind capital case if that's the convention used in the struct.